website/components/live-example/Knobs.tsx

```diff
@@ -244,34 +238,34 @@ function Area({ onChange, label, value, prop, darkMode }: TextInterface) {
   const labelId = useMemo(() => areaIdAllocator.generate(), []);
 
   return (
-    <div
-      className={cx(
-        knobContainerStyle,
-        css`
-          border-top: 1px solid
-            ${darkMode ? uiColors.gray.dark2 : uiColors.gray.light2};
-        `,
-      )}
-    >
+    <KnobRow darkMode={darkMode}>
```

Comment:
nit: should we just turn the composed classname into a `getClassName` function rather than adding another component?

Reply:
I don't like getClassName functions in general 😄 I'm applying the same feedback I gave on one of your recent PRs here,